src/Validation/Validation.php

```diff
@@ -1431,6 +1431,17 @@ public static function isArray($value)
     }
 
     /**
+     * Check that the input value is a 6 digits hex color.
```

Comment:
@inoas https://developers.whatwg.org/common-microsyntaxes.html#simple-color
> ... the remaining six characters are all ASCII hex digits ...

but I am fine to add an additional option or change the method name if you want.